examples/custom.ad.example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "1": {
+    "src": "https:\/\/bachtrack.com\/files\/46901-banner-728x90px-uk-2016-5sec.gif",
```

Comment:
just want to make sure, it these escape character necessary? how can we prevent them.